Makefile

```diff
@@ -15,8 +15,13 @@ deep-clean:
 
 # Run the application
 run:
+	cp .env.example .env
 	docker-compose up
 
+# Launch the application, open browser, no stdout
+run-launch:
+	bin/launch.sh
+
 # Open a bash shell on the running container
 bash:
 	docker-compose exec wordpress bash
```

Comment:
Noticed this is missing the "-it" flag, so that it actually opens in the interactive shell, could you add that in while you're at it?

Reply:
Thanks for letting me know about this! As we discovered; docker-compose wraps the interactive element of the exec command for ease allowing us not to define it